Testing/contracts/abiTypes.ts

```diff
@@ -0,0 +1,88 @@
+import BN from 'bn.js'
+
+export interface IComplexContract{
```

Comment:
Got a lot of re-used union types here, you could use a type union assigned to an alias to make it more legible